save-common/src/commonMain/kotlin/com/saveourtool/save/core/config/EvaluatedToolConfig.kt

```diff
@@ -1,16 +1,12 @@
 package com.saveourtool.save.core.config
 
 /**
- * Configuration for an evaluated tool, that is read from test suite configuration file (toml config) or cli
+ * Configuration for an evaluated tool, that is read from cli
  *
- * @property execCmd
- * @property execFlags
  * @property batchSize it controls how many files execCmd will process at a time
  * @property batchSeparator A separator to join test files to string if the tested tool supports processing of file batches (`batch-size` > 1)
  */
 data class EvaluatedToolConfig(
-    val execCmd: String?,
-    val execFlags: String?,
```

Comment:
What about those properties? They are removed because they can be set either from save.toml or from ~save-overrides.toml~ CLI flags? Isn't it overcomplication to configure the evaluated tool from three places?

Reply:
`execCmd` and `execFlags` now can be set only in `save.toml` (in future it will be combined to a single field) -- personal tests and can be overridden by `save-overrides.toml` -- public\sharable tests

Reply:
Then what's the logic behind keeping batch size a a CLI flag? Maybe we should move everything to the previous logic with TOML files only?

Reply:
actually, I partially agree: I personally think that we should extract all configs are related to tool from `save.toml`, but I understand the use case with private tests when execCmd is stored inside for ease usage and the case when execFlags changes from one test suite to another test suite.

@akuleshov7, what do you think? maybe return `batchSize` and `batchSeparator` back to `save.toml` but to general section?
Motivation: we don't remove `execCmd` and `execFlags` from `save.toml`, let's store `batchSize` and `batchSeparator` there

Reply:
discussed -- agreed to move to `GeneralConfig`

Reply:
totally agree, I would even remove all non-critical arguments from cli